modules/level_set/src/postprocessors/LevelSetCFLCondition.C

```diff
@@ -0,0 +1,55 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "LevelSetCFLCondition.h"
+
+template<>
+InputParameters validParams<LevelSetCFLCondition>()
+{
+  InputParameters params = validParams<ElementPostprocessor>();
+  params.addClassDescription("Compute the minimum timestep from the Courant–Friedrichs–Lewy (CFL) condition for the level-set equation.");
```

Comment:
Courant–Friedrichs–Lewy 
The dashes are unicode 2013. Do we allow unicode in parameter descriptions?
This is currently breaking the syntax output for modules.
We could fix it on the syntax generation side as well in `genInputFileSyntaxHTML.py` by first encoding to utf-8